index.js

```diff
@@ -0,0 +1,7 @@
+import ADBDevice from './lib/adb-device';
+
+function connectToDevice (type) {
```

Comment:
There's no reason they can't.  The whole exporting modules into an `index.js` file and then creating  another file to run the modules is strange to me coming from C/C++, so if I'm doing it incorrectly then please let me know.